docs/painless/painless-contexts/painless-watcher-context-example.asciidoc

```diff
@@ -0,0 +1,112 @@
+==== Complete Example
+
+While these examples are contrived, a more complete watch shown below contains
+both <<painless-watcher-condition-context, conditions>> and
+<<painless-watcher-transform-context, transforms>>, and is the basis of the
+above examples. It is shown for completeness.
+
```

Comment:
Maybe: The following example shows scripted watch and action conditions within the context of a complete watch. This watch also uses a scripted <<painless-watcher-transform-context, transform>>.

I wouldn't link back to the top of this topic.